src/wing/CCPACSWing.cpp

```diff
@@ -858,9 +860,19 @@ CCPACSGuideCurve& CCPACSWing::GetGuideCurveSegment(std::string uid)
 
 TopoDS_Compound CCPACSWing::GetGuideCurveWires() const
 {
-    return *guideCurves;
+    return guideCurves->wiresAsCompound;
 }
 
+std::vector<double> CCPACSWing::GetGuideCurveStartParameters() const
```

Comment:
This is duplicate to CCPACSGuideCurves->GetParamters

Reply:
On the other hand, reaching out to a specific segment->guidecurves->parameters is not very handy.

Reply:
Still, we now have two functions, doing *almost* the same thing in different ways. The result of CCPACSGuideCurves->GetRelativeCircumferenceParameters does this

https://github.com/DLR-SC/tigl/blob/2d5c6d4a683e7c4b0ba15483d8bdf34d22318f50/src/guide_curves/CCPACSGuideCurves.cpp#L107-L110

while CCPACSWing::GetGuideCurveStartParameters does not.

Reply:
As I have written above, we should use only the code from CCPACSGuideCurves and remove the wing function. Even though accessing the guide curves parameters from the wing is ugly. We could still keep the function but call the GuideCurves function from it.

Reply:
> We could still keep the function but call the GuideCurves function from it.
I agree